melodic/distribution.yaml

```diff
@@ -5481,6 +5481,13 @@ repositories:
       url: https://github.com/astuff/ml_classifiers.git
       version: master
     status: maintained
+  mobile_robot_simulator:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/nobleo/mobile_robot_simulator-release.git
```

Comment:
The source repository https://github.com/nobleo/mobile_robot_simulator/blob/master/package.xml says that the license is BSD, but there is no LICENSE file in the repository.  Please add one there.

Also consider adding doc and source blocks to this entry, though that is not required.